src/Admin/components/Dashboard/Dashboard.js

```diff
@@ -37,8 +44,86 @@ const Dashboard = ({
 
           <div className='Listing'>
             <Tabs defaultActiveKey='ongoing'>
-              <Tab eventKey='ongoing' title='Ongoing'></Tab>
-              <Tab eventKey='previous' title='Previous'></Tab>
+              <Tab eventKey='ongoing' title='Ongoing'>
+              {dashboardInfo[0] &&
+                dashboardInfo[0]?.ongoing.length === 0 ? (
+                  <Container>
+                    <h4 style={{ color: "#787878" }}>No Listings Available</h4>
+                  </Container>
+                ) : (
+                  <Fragment>
+                    {dashboardInfo[0]?.ongoing.map((elem) => {
+                      
+                      return (
+                        <Cards
+                          key={elem.id}
+                          id={elem.id}
+                          token={token}
+                          company={elem.company_details.name}
+                          compensation={elem.compensation}

```

Comment:
elem.compensation_CTC instead of elem.compensation

Reply:
done